Frontend/src/components/usermanagement/SettingsDeleteUser.vue

```diff
@@ -67,28 +72,49 @@ export default {
           headers: {
             'Authorization': `Bearer ${this.$store.state.user.token}`
           }
-        }).then(()=> {
-          console.log("Alle Photos erfolgreich geloescht")
+        }).then((response)=> {
+          if(response && response.status == 200){
+            const answer = 'All you pictures deleted successfully!';
```

Comment:
Localization müsste gier gar nicht so kompiziert sein. In der Search New Freinds komponente habe ich auch eine String Variable mit Localization implmentiert:  this.feedback = 'error.friendAlready';